### DISCUSSION TITLE
Edge Functions - Import from files outside supabase/functions directory

### DISCUSSION THREAD

**Discussion by cohlar** [supabase/supabase#11517] [Questions]
Hi there,

I'd like to know if there is a way to import local resources to edge functions from outside the `supabase/functions` directory.

Specifically, I have a `models` root directory with TypeScript types and classes that I'd like to share across my frontend and my edge functions.

I have tried a relative import, but get a `Module not found` error.

![image](https://user-images.githubusercontent.com/57757008/211181077-a2438d99-e493-4398-a53c-241d146f2f24.png)

-- [reposting from Discord for traceability](https://discord.com/channels/839993398554656828/1061507682474926162/1061507682474926162).

**Comment by SteveChurch** [comment]
AFAIK you can only access files within the `functions` directory. These are compiled (I believe) 

https://supabase.com/docs/guides/functions/quickstart#organizing-your-edge-functions
